trading_calendars/exchange_calendar_xmad.py

```diff
@@ -63,11 +71,19 @@
 
 ImmaculateConception = immaculate_conception(end_date='2005')
 
-ChristmasEve = christmas_eve(end_date='2011')
+ChristmasEveThrough2010 = christmas_eve(end_date='2011')
+ChristmasEveEarlyClose2012Onwards = christmas_eve(
+    start_date='2012',
+    days_of_week=(MONDAY, TUESDAY, WEDNESDAY, THURSDAY, FRIDAY),
```

Comment:
you could do `days_of_week=WEEKDAYS` here and below, `WEEKDAYS` is in `trading_calendar.py`